docs/stratis.txt

```diff
@@ -326,14 +329,19 @@ Allows Overprovisioning::
 Encryption Enabled:
 	Whether or not encryption is enabled for this pool.
 
-Encryption Enabled: Key Description::
+Encryption Enabled: Key Description (V1 only)::
```

Comment:
I believe this is V2 only actually. In V1, we have the exact encryption information.

Reply:
Disregard these. I thought this was referencing the boolean of whether any key description was enabled.